pkg/pillar/scripts/generate-device.sh

```diff
@@ -5,15 +5,22 @@
 #
 # Generate a self-signed ECC certificate; 20 year lifetime
 # argument is a basename for the output files
-# Example: generate-dc.sh ../run/test
+# Example: generate-device.sh ../run/test
 # will place the private key in ../run/test.key.pem and the cert in
 # ../run/test.cert.pem
+# Example: generate-device.sh ../run/test use_tpm
+# will use TPM to generate the key pair, and will place just cert in
+# ../run/test.cert.pem
 
-if [ $# != 1 ]; then
+if [ $# -lt 1 ]; then
     myname=$(basename "$0")
-    echo "Usage: $myname <output basename>"
+    echo "Usage: $myname <output basename> [use_tpm]"
```

Comment:
Makes sense to use -t (to follow getopt conventions) instead of a "use_tmp" form of argument.
Hence usage becomes $myname [-t] <output basename>

Reply:
Addressed.